### ISSUE TITLE
[Other - Accessibility] No warning given for link opening in new tab. (00.00.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#56081]
#### VFS team name
Benefits Team 1

#### VFS product name
Supplemental Claims Form 20-0995

#### VFS feature name
Supplemental Claims Form 20-0995

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Step 3 of 4: New and relevant evidence - Review and acknowlede the notice of evidence needed, the "Review the notice of evidence needed" link opens in a new tab. Screen reader users should be warned about this change of context, since opening in a new tab can sometimes be disorienting (might expect the back button to work, not clear how to navigate back).

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
If possible, it would be great to have a visible text warning for all users, eg.
```
<a href="/disability/how-to-file-claim/evidence-needed/" target="_blank" rel="noopener noreferrer">Review the notice of evidence needed (opens in a new tab)</a>
```
If that's not practical for design reasons, then an `aria-label` just for screen reader users would be sufficient.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by eileen-coforma** [comment]
Change link to read 
`Review the notice of evidence needed (opens in new tab)`
[SKETCH](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/a/GoO9zkE)

**Comment by Mottie** [comment]
Available for testing in staging & production

**Comment by saderagsdale** [comment]
@joshkimux Hey! Can you check this one when you have a sec?

**Comment by joshkimux** [comment]
<img width="613" alt="Screenshot 2023-04-14 at 1 08 10 PM" src="https://user-images.githubusercontent.com/14154792/232111161-18c8b452-3773-4623-9ac8-b24e6ac51553.png">

Perfect! Closing this out 😄
